README.md

```diff
@@ -115,6 +115,19 @@ A full list of all the unit tests we run in our CI can be found [here](.github/s
 MMTk is also tested with the VM bindings we are maintaining by running standard test/benchmark suites for the VMs.
 For details, please refer to each VM binding repository.
 
+## Supported Rust versions
+
+As a research project, MMTk is quite aggressive in following the latest releases of the Rust
+toolchain. The minimum supported Rust version (MSRV) is "N-1", that is, one minor version before the
+latest stable Rust release.  For example, if the latest Rust stable release is "1.61.2", then the
+MSRV will be "1.60.0".
+
+MMTk has a six-week release cycle, roughly the same as Rust itself.  We bump the MSRV version
```

Comment:
>   Last time we talked with @caizixian on this. We agreed on bumping the version in rust-toolchain for every other release (he needs to update our dev and CI machines).

If updating the toolchains every 6 weeks is too much work, we can just say our MSRV is "N-2".  I am still OK with "N-2" or "N-3".  In the case of "N-2", MMTk-core will support three Rust stable versions, and each Rust version remain supported for 18 weeks.  If we update our toolchains for every even (or odd) Rust releases (that is, every 12 weeks), the existing installed toolchain will remain supported by the time we do our update.  In other word, on the day when an even Rust version (assume we update Rust at even versions) is released, both the currently installed toolchain and the newest toolchain will be supported, and we can do the update in the window of 6 weeks from that day until the next Rust release.  (Actually we may have up to 12 weeks because the MSRV of MMTk core is the "N-2" right after the *previous* MMTk core release.)

@caizixian What do you think of it? If "N-2" is still too much work, we can use "N-3" instead.